github/projects.yaml

```diff
@@ -124,3 +124,8 @@
   description: Ansible collection for network devices
 - project: ansible/workshops
   description: Training Course for Ansible Automation Platform
+- project: project-receptor/receptor
```

Comment:
Thanks for adding this, right now my account in github (pabelanger) needs to be admin in the project namespace, this is because we configure some repo settings zuul.

I plan to switch from my personal account to 'ansible-zuul' user, but haven't done so yet.

Reply:
@matburt Can you take care of the above ^ (and eventually add me too in case they are more projects coming in under the `project-receptor` umbrella)

Reply:
done!